internal/field/internal/templates/element/tests.go

```diff
@@ -10,7 +10,7 @@ import (
 	"math/bits"
 	"fmt"
 	{{if .UsingP20Inverse}} 
-	"github.com/consensys/gnark-crypto/field"
+	"github.com/consensys/gnark-crypto/internal/field"
```

Comment:
Because of this, generated files can not be embedded in another project as a standalone library.